app/src/main/res/layout/profile_progress_header.xml

```diff
@@ -51,121 +49,133 @@
       android:fontFamily="sans-serif-medium"
       android:text="@{viewModel.profile.name}"
       android:textColor="@color/oppiaPrimaryText"
-      android:textSize="18sp"
+      android:textSize="20sp"
       app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toBottomOf="@+id/profile_frame_layout" />
+      app:layout_constraintTop_toBottomOf="@+id/profile_edit_image" />
 
-    <LinearLayout
-      android:id="@+id/progress_stats_linear_layout"
-      android:layout_width="wrap_content"
+    <com.google.android.material.card.MaterialCardView
```

Comment:
There is border on both the cards for which you can use `grey_card_rounded_border.xml` file.

Reply:
This border has been used in other items of `OngoingTopicListActivity` and `CompletedStoryListActivty`

Reply:
Ditto for landscape layout file

Reply:
Done